exercises/practice/grade-school/uGradeSchool.pas

```diff
@@ -0,0 +1,54 @@
+                                              unit uGradeSchool;
+
+interface
+
+uses
+  Generics.Collections;
+
+type
+  TRoster = TList<string>;
+  TGrades = TDictionary<Integer, TRoster>;
+
+  ISchool = interface
+    procedure Add(const aName: string; aGrade: integer);
+    function Grade(aGrade: integer): TRoster;
+    function Roster: TRoster;
+  end;
+
+  TSchool = class(TInterfacedObject, ISchool)
+  private
+    procedure Add(const aName: string; aGrade: integer);
+    function Grade(aGrade: integer): TRoster;
+    function Roster: TRoster;
+  end;
```

Comment:
This should be moved to the implementation section above `NewSchool`.  Also, `NewSchool` should probably be completed instead of stubbed.